tests/conformance/common.go

```diff
@@ -577,8 +576,10 @@ func loadStateStore(tc TestComponent) state.Store {
 		store = s_awsdynamodb.NewDynamoDBStateStore(testLogger)
 	case "aws.dynamodb.terraform":
 		store = s_awsdynamodb.NewDynamoDBStateStore(testLogger)
-	case "etcd":
-		store = s_etcd.NewEtcdStateStore(testLogger)
+	case "etcd.v1":
```

Comment:
Does this need to be added to the test_info.mjs file?

Reply:
I think so. Thanks for the catch. Will update the file to test both